package.json

```diff
@@ -14,25 +14,27 @@
   },
   "license": "BSD-3-Clause",
   "author": "The WebRTC project authors",
-  "main": "Gruntfile.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/webrtc/samples.git"
   },
   "scripts": {
-    "test": "grunt && start-tests"
+    "test": "gulp && start-tests"
   },
   "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-preset-es2015": "^6.24.1",
     "eslint-config-google": "^0.9.1",
     "express": "^4.14.1",
     "geckodriver": "^1.10.0",
-    "grunt": "^0.4.5",
-    "grunt-cli": ">=0.1.9",
-    "grunt-contrib-compress": "^0.13.0",
-    "grunt-eslint": "^21.0.0",
-    "grunt-githooks": "^0.3.1",
-    "grunt-htmlhint": ">=0.9.12",
+    "gulp-csslint": "^1.0.1",
+    "gulp-eslint": "^5.0.0",
+    "gulp-htmlhint": "^2.1.1",
+    "gulp-zip": "^4.2.0",
     "pem": "^1.9.4",
     "webrtc-utilities": "^1.0.8"
+  },
+  "dependencies": {
+    "gulp": "^3.9.1"
```

Comment:
devDependency. Not this isn't published to npm so it does not really matter but... :-)